contrib/helm-charts/Portus/templates/portus-secret.yaml

```diff
@@ -0,0 +1,93 @@
+    SVBsL0h6TFByeE9xSWQ1bmhVUmxNNgo0ZWg1U2krUFBOVnZlRFdlODJKV2tUdHJnNHNlVDBrRVNV
+    a3BqMGRqMm9tOERjUHgrOE1ZTUNvN1NLWkl2YjhTCkJRM0hqRFZwZUhVVDhOcXlzYXVRcFZqQWla
+    WmE5Q2YzYzNvaXpyeE5KVEthdlF1TnJUKzVOaW9PRmZPZDRIS1YKOWFCOVNNOG42ODc3dUpvTEFn
+    TUJBQUVDZ2dHQWFwUGk2R0FGMlBNSnVWSWowRGk1UnRRbXVXb0U5YmEzYldNaQo1MFVJR3Zsd1Vj
+    MmFva2MraDNsdWxWYy9wTW13eWlLZU1ZRlpzbzIxcHRPK3Z4Uk9uWk1PeVhTWnJzNlUyalhWCk5l
+    U0xsTDIzcDRZOWRtSkpKNzVWVUVYNU96MmVKTmFCUStYdTJtUmFmNHdFbUY2UFRwK2xSN2pUWUVo
+    VWorMU4KQ3FUdFludC91QUZKZ2JCcnVrRFhXV1lTQTdJblBneVFWelVubzl1eDlNWFo0SUhacm9q
+    TVl5TG12ZUJrVjIvaQp0aHZTYXk3M3d5TFVUUG95MmZXRnJnQnhwYkJCR1FQWmhwSDlhNGU0Wkdn
+    QVVMU25pamlteGlOVWVNMk9XdllUCnJ4UWpRZjRmL2JZdEZzNHdiZUppVzNmajhNajZHVUNOamNj
+    UG5MR1VSWFNRQ2ljdEdXOUYxZnJrMVNIODFSTTQKZ1FIR3YwVUpUYmY3RlVFSzZHSEdqeVg4dFlH
+    cmU2TGxUcmlFczNyM3VkaUxXUDUva3pTYXY1N1dVYW5yMlY5UwpzOWlOYmFrMkZ0WHp3S3d0eWQv
+    R0hoYVF5UUVYbE5JbytCbEt0NTFkdHpzOXpxRUtvNk5WUXNOMUxFT0RURTB6CjIyMEVqTlhYdUxz
+    TmJPSm1XUVFUbWd0cWpCczVBb0hCQVBQUnRGdS85MnVGd3JWRnN1SVp1USsyRGF5RmYveS8KNXBW
+    RWFjNnFrRkxtMDZ0azVIeFlmS3d1TnR5eGxlZlh2UDRhU05wQTBzOW04YVNIQlpGOXF3QWhvdHdK
+    TU1RYQpMeGYvMTJQeUQyM1hvc2tJejR1M0p2UTh0WGNWR1lpOVUrbk9xV0FucEp0aVRocC9DUXd2
+    L21QNGlVanlXcS9JCmZqM0drNjlNL0pFZTJtS0pzMmRtR2ZNZ2hDRE91VCt1RTA3WlJITlN4NVR1
+    cCtzTFZCSWhpdmN0TVRoVEx5YnoKOHpzaHg2NDVNRHJxM3ZhMTBCYXlXakl0OHJIV3J5N0JId0tC
+    d1FEWGRxY3Q2cXVaMjd4MEZnbnhnM01ZY3pIawpqSHJXa050OUpvcWFEL2R4c0hiWWt6bEszY05D
+    bk1ON3lsUUhrc1M3cE5jNlA0Nmo5SEtNWUVVSkp6ajd2dkh3CllhMkN1R0ZRRDBpV3YyYlVpZW9y
+    MVd1cWRhU3EyNitNZ3JyNnVmQS9FbzlmZURDUXFQTnQ5Szk3RnVTWG10VmsKZGFYcGJJbVU0YkJW
+    K0FEaUg5dXZPbEJyS0hJUkt6Z0wvUmozb2xOTlREUUpCOXFySEtUNVV6d0lOS1BBSTF4Wgo5SWht
+    OTNBMVBJUWRjUFg0UWpudTlYV1JNUy9pNndnWklkRzRiWlVDZ2NBUEtCZ1lKQVRUY0VzOFRZcmpO
+    MG9qCnllTnJ1QmpPczNjNEREcXZtRGJFa0lqVC9MK2ZZempNMDBxaHlZQlNHejBxWnFQa3ZXYmRj
+    YjdQeFh6NUlHREMKNzFSYWJVL1V3UFB3Zi96NXY1QnArWitxODV0RFNpMW52V1ppY3JOVnByOVNw
+    NEFyOUFydGhkenBGTUNSTzJLTApBR1k0WEtoNmlQSWdFUG9aU2FYaHQ1TkgwTnVRTCs2cENIcHVO
+    NnVZWTgvYmRQSW1ZUno3Q3lMeEg1RW9WMkYyCklCOFJLMHFoRjBVbGhlSU4ySXhBL1Z6QzVXWjhJ
+    cllzckdLTDM1YWdSMjBDZ2NFQWkvY3pJTnZaLzRmRHd6RngKNGliL0JIb0loaXhQdThTOElHK0ND
+    QlNwTEh4eXA3dnpzdE9lZlU0eW10N1RlVnY1V3QvNVJJaDNFOEMvREx1WQovUm95cFM3a0ljQ0NJ
+    M0c5SFhIZ09MVTdpb1BRMEFyQzl5ZVhBYnJXTGhQYWZQZCt2cXBDNHNURDM2VXhCVk8zCkR1SFBP
+    MERFc0k3UWFYTXRhTDYzZXhFeEdiT1NWU0V2am56bXBJcUx6UlhzT3ltQ3Jhcjcvb1hRb2Vsb3Jl
+    SkoKRkJlRjdGSE1NMDlyc0YrNno0cHhWR0xHRmpDSFM5aStOOFJaRkRLVHZ5ZStBZTJGQW9IQkFP
+    MDV5dVgvOHM0OQpXQ1dpUmM2R2tNd254bkVQektUT3VSKzM5K1Vja0JQOFovS3F3L3pYc2pySmZt
+    YU12eTZpazFIMm5vVWhQSTFOCkxKQW1FVVFlaHhoR0JFRWUvckJxYzVOZUNteWJUL0oyd244TVpW
+    Y0JuaWNGb2ZFRGtXWXNKYzBTaWg1U1d5NWYKZEpGc1d1eThmN1p6bUpjWFZrdTVQQ2JycUhSWjU4
+    Mk9ueVI4Wkp0N0wrWnRRWkNkU1R5VFAxcmVNMGhsbEhEbQpuTFg0dzE2UTZybThtSUVzWC9ydzJX
+    SWl5Y21PQW9MUGU4NGJiMFhJN2lzelhITENHK0VVakE9PQotLS0tLUVORCBSU0EgUFJJVkFURSBL
+    RVktLS0tLQo=
```

Comment:
We should probably make the key (and cert) come from the Values file instead of having these directly in the templates.